install_requirements.py

```diff
@@ -129,14 +145,25 @@ def python_is_compatible():
     "zstd",  # Imported by resolve_buck.py.
 ]
 
-# pip packages needed to run examples.
-# TODO: Make each example publish its own requirements.txt
-EXAMPLES_REQUIREMENTS = [
-    "timm==1.0.7",
-    f"torchaudio==2.5.0.{NIGHTLY_VERSION}",
-    "torchsr==1.0.4",
-    "transformers==4.42.4",
-]
+if USE_PYTORCH_NIGHTLY:
```

Comment:
Isn't it cleaner to have a single if block for the CI? And it should only apply to torch and the domain libraries, i.e. torch, torchvision, torchaudio?